tests/metadata-service-tests/test/testkit/k8sresources_check.go

```diff
@@ -3,7 +3,7 @@ package testkit
 import (
 	"testing"
 
-	istio "github.com/kyma-project/kyma/components/gateway/pkg/apis/istio/v1alpha2"
+	istio "github.com/kyma-project/kyma/components/application-connector/pkg/apis/istio/v1alpha2"
```

Comment:
The path on this branch is `.../metadata-service/pkg/...`, please update this after the changes are merged.